Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -549,7 +549,7 @@ class BrowserViewController: UIViewController {
             self.view.alpha = (profile.prefs.intForKey(IntroViewControllerSeenProfileKey) != nil) ? 1.0 : 0.0
         }
 
-        if !displayedRestoreTabsAlert && hasPendingCrashReport() {
+        if !displayedRestoreTabsAlert && !cleanlyBackgrounded() && crashedLastLaunch() {
```

Comment:
We're still relying on it (via Sentry) to let us know if there's a pending crash report (meaning we crashed during the last run). The only difference is now we're also using `cleanlyBackgrounded` to ignore any pending crash reports if the crash happened after the app went to the background.